bin/stacks/routing-lambda-stack.ts

```diff
@@ -117,7 +119,7 @@ export class RoutingLambdaStack extends cdk.NestedStack {
 
       description: 'Routing Lambda',
       environment: {
-        VERSION: '23',
+        VERSION: '24',
```

Comment:
this is not really necessary since we are setting the tenderly key as a new key here now, but it doesnt harm either.